src/components/ContentModal.vue

```diff
@@ -0,0 +1,81 @@
+<template>
```

Comment:
Sure you want to build your own modal component? It is ok, but also means much more effort, whereby there are already many good existing modal components. In general, I just noticed that you want to do the style and components completely yourself.

Reply:
Initially I was not sure if we can use prebuilt components. So I decided to build one myself.